build/params_shared_vals.go

```diff
@@ -1,85 +1,55 @@
-//go:build !testground
-// +build !testground
-
 package build
 
 import (
 	"math/big"
-	"os"
-
-	"github.com/filecoin-project/go-address"
-	"github.com/filecoin-project/go-state-types/abi"
-	"github.com/filecoin-project/go-state-types/network"
-	builtin2 "github.com/filecoin-project/specs-actors/v2/actors/builtin"
 
-	"github.com/filecoin-project/lotus/chain/actors/policy"
+	"github.com/filecoin-project/lotus/build/buildconstants"
 )
 
 // /////
 // Storage
 
-const UnixfsChunkSize uint64 = 1 << 20
-const UnixfsLinksPerLevel = 1024
+var UnixfsChunkSize uint64 = buildconstants.UnixfsChunkSize  // Deprecated: Use buildconstants.UnixfsChunkSize instead
+var UnixfsLinksPerLevel = buildconstants.UnixfsLinksPerLevel // Deprecated: Use buildconstants.UnixfsLinksPerLevel instead
 
 // /////
 // Consensus / Network
 
-const AllowableClockDriftSecs = uint64(1)
-
-// Used by tests and some obscure tooling
-/* inline-gen template
-const TestNetworkVersion = network.Version{{.latestNetworkVersion}}
-/* inline-gen start */
-const TestNetworkVersion = network.Version23
-
-/* inline-gen end */
+var AllowableClockDriftSecs = buildconstants.AllowableClockDriftSecs // Deprecated: Use buildconstants.AllowableClockDriftSecs instead
 
 // Epochs
-const ForkLengthThreshold = Finality
+var ForkLengthThreshold = Finality // Deprecated: Use Finality instead
```

Comment:
Why is this a `var` now? And is there a good reason for deprecation of this? (Asking from a position of ignorance, but it is used throughout the code).